ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -394,13 +394,13 @@ public void handleDisconnect(
       final DisconnectReason reason,
       final boolean initiatedByPeer) {
     if (ethPeers.registerDisconnect(connection)) {
-      LOG.debug(
-          "Disconnect - {} - {} - {}... - {} peers left\n{}",
+      LOG.info(
+          "Disconnect - {} - {} - {}... - {} peers left",
           initiatedByPeer ? "Inbound" : "Outbound",
           reason,
           connection.getPeer().getId().slice(0, 8),
-          ethPeers.peerCount(),
-          ethPeers);
+          ethPeers.peerCount());
+      LOG.trace(ethPeers.toString());
```

Comment:
small opt

```suggestion
      LOG.trace(ethPeers);
```

Reply:
I had to do this to get it to compile - is this better?

      LOG.atTrace().addArgument(ethPeers).log();

Reply:
Ok is see the problem with my initial suggestion, it should be `LOG.trace("{}", ethPeers);`